.github/workflows/develop.yml

```diff
@@ -31,7 +31,7 @@ jobs:
         id: cache
         with:
           path: ~/.cache/pip
-          key: tests-${{ runner.os }}-${{ matrix.python-version }}-pip-${{ hashFiles('requirements/main.txt', 'requirements/dev.txt') }}
+          key: tests-${{ runner.os }}-${{ matrix.python-version }}-pip-${{ hashFiles('requirements/core.txt', 'requirements/dev.txt') }}
```

Comment:
The test cases need more than just the core requirements, right? (Since they touch all models)

Reply:
Right, that's corrected.